index.js

```diff
@@ -4,12 +4,54 @@ const ecstatic = require('ecstatic');
 const fs = require('fs-extra');
 const glob = require('glob');
 
+const createEnvFile = async ({ inputs, builtPages }) => {
```

Comment:
This main changes in this PR are running almost everything inside `__dirname` instead of `cwd`.